curp/src/client.rs

```diff
@@ -504,9 +504,12 @@ where
         Err(ProposeError::Timeout)
     }
 
-    /// Send fetch leader requests to all servers until there is a leader
+    /// Send linearizable fetch leader requests to all servers until there is a leader
     /// Note: The fetched leader may still be outdated
```

Comment:
Please update "fetch leader" to "fetch cluster"

Reply:
done